src/Core/src/Handlers/Picker/PickerHandler.iOS.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+using System.Collections.Specialized;
+using UIKit;
+using RectangleF = CoreGraphics.CGRect;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class PickerHandler : AbstractViewHandler<IPicker, NativePicker>
+	{
+		UIPickerView? _pickerView;
```

Comment:
Just found this on the Apple docs. Probably not essential for now, maybe...

> **Important**
> UIPickerView and its descendants aren’t available when the user interface idiom is UIUserInterfaceIdiomMac.

https://developer.apple.com/documentation/uikit/uipickerview